### ISSUE TITLE
Could not read the Face detection model

### ISSUE THREAD

**Issue by photonicpiston** [achen353/Face-Mask-Detector#3]
Hey when I run the code in 'detect_mask_video', I receive this error: 

 cv2.error: OpenCV(4.5.5) D:\a\opencv-python\opencv-python\opencv\modules\dnn\src\caffe\caffe_io.cpp:1126: error: (-2:Unspecified error) FAILED: fs.is_open(). Can't open "./face_detector_model/deploy.prototxt" in function 'cv::dnn::ReadProtoFromTextFile' 

My installed libraries are: 

Tensorflow~=2.7.0
matplotlib~=3.4.1
numpy~=1.19.5
scikit-learn~=0.24.1
opencv-python-headless~=4.5.5.62
dash~=1.18.1
textwrap3~=0.9.2
gunicorn~=20.0.4

Is this happening because my libraries aren't the same versions?
